csharp/OnnxRuntime.CSharp.proj

```diff
@@ -105,15 +105,15 @@ CMake creates a target to this project
 
     <Copy
     SourceFiles="$(NativeBuildOutputDirAbs)\$(OrtPackageId).$(PackageVersion).nupkg"
-    DestinationFolder="$(NativeBuildOutputDirAbs)\nuget-artifacts"
+    DestinationFolder="$(NativeBuildOutputDirAbs)\nuget-local-artifacts"
     />
```

Comment:
There are some CI files that I think use the nuget-artifacts directory. Do those need updating?

\tools\ci_build\github\azure-pipelines\templates\c-api-cpu.yml
\tools\ci_build\github\windows\extract_nuget_files.ps1
\tools\ci_build\github\window\extract_nuget_files_gpu.ps1

Same question for generate_nuspec_for_native_nuget.py

Reply:
No.
The CI files use this directory to store all the downloaded artifacts from different jobs which the packaging pipeline depends on. "generate_nuspec_for_native_nuget.py" script then uses the existence of this directory to determine whether this is ADO pipeline build or local build.

This step merely copies the native nuget package to this "nuget-local-artifacts" dir for better discovery. After this step there are 2 copies of the native package: 1. in <native_build_dir>\{config}\{config} and 2. <native_build_dir>\{config}\{config}\nuget-local-artifacts 

We can delete this copy step altogether.